blocks/editable/format-toolbar/style.scss

```diff
@@ -41,3 +41,11 @@
 		margin: 0;
 	}
 }
+
+.blocks-url-input {
+	input[type=url] {
+		&::-ms-clear {
```

Comment:
I don't think we need so much nesting here, particularly when using the `&` and there being no other style selectors. As written, this could be simply:

```css
.blocks-url-input input[type=url]::-ms-clear {
	display: none;
}
```

Or when moved to `blocks/url-input/style.scss`:

```scss
.blocks-url-input {
	// ...

	input[type=url]::-ms-clear {
		display: none;
	}

	// ...
}
```